src/GitVersion.BuildAgents/Agents/BitBucketPipelines.cs

```diff
@@ -10,6 +10,7 @@ internal class BitBucketPipelines : BuildAgentBase
     public const string TagEnvironmentVariableName = "BITBUCKET_TAG";
     public const string PullRequestEnvironmentVariableName = "BITBUCKET_PR_ID";
     private string? file;
```

Comment:
please rename file to `propertyFile` to easier distinguish

Reply:
Fixed.